interp/cfg.go

```diff
@@ -507,9 +507,13 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 					src.findex = dest.findex // Set recv address to LHS
 					dest.typ = src.typ
 				case n.action == aAssign && src.action == aCompositeLit:
-					n.gen = nop
-					src.findex = dest.findex
-					src.level = level
+					if !(dest.typ.cat == valueT && dest.typ.rtype.Kind() == reflect.Interface) {
```

Comment:
A common practice in Go is to "return early" as much as possible, to avoid extra indentation, and hence improve readability. So you could invert that if, and unindent the whole block, i.e.:
```
if (dest.typ.cat == valueT && dest.typ.rtype.Kind() == reflect.Interface) {
    break
}
n.gen = nop
...
```